### ISSUE TITLE
Making voices can disrupt makeAccidentals()

### ISSUE THREAD

**Issue by jacobtylerwalls** [cuthbertLab/music21#577]
Based on case reported by Igor Praznik, music21 list 3/3/20.

```
s = stream.Stream()
s.insert(key.Key('Gb'))
s.insert(0, note.Note('D-5'))
s.insert(0, note.Note('D-4'))
s.show()  # During export to xml, makeNotation(), makeVoices(), makeMeasures()
```
<img width="138" alt="Screen Shot 2020-07-14 at 9 19 28 PM" src="https://user-images.githubusercontent.com/38668450/87492057-b5828a00-c617-11ea-86ca-9e4ea46d96c3.png">

`makeNotation()` calls `makeAccidentals()` per `Measure` object, but `makeAccidentals()` only inspects `measure.notesAndRests`, which is empty in this case because the notes have been put in voices already:

https://github.com/cuthbertLab/music21/blob/93bea7cc87bdb9fb2bb0ab0cb3faef1717b24004/music21/stream/__init__.py#L5867

<s>Need to grab the voice objects somewhere, possibly just recursively calling `makeAccidentals()`, but there are going to be corner cases here with accidentals tied from previous measures.</s> Update: I think this is only affecting the code path where measures have not been created yet. In that case, probably easier to just call `makeAccidentals()` before `makeVoices()` and `makeMeasures()`, since without measures to worry about, those corner cases go away.

**Comment by jacobtylerwalls** [comment]
Or, could even do this in `makeVoices()`, since the root of the problem is that notes are being yanked from `Measure` objects into `Voices`, something like:

```
if not returnObj.haveAccidentalsBeenMade():
    returnObj.makeAccidentals()
```

**Comment by mscuthbert** [comment]
Hmmm...it seems better to fix this in makeAccidentals or makeNotation, since I wouldn't expect makeVoices to affect accidental status.  But not entirely sure.

**Comment by jacobtylerwalls** [comment]
Agreed. The patch I linked just adds recurse() in makeAccidentals.

**Comment by mscuthbert** [comment]
Fixed in #578  -- thanks!
